dll/win32/shell32/utils.cpp

```diff
@@ -297,3 +297,35 @@ LargeIntegerToString(
     return Int64ToString(pLargeInt->QuadPart, pszOut, cchOut, bUseFormat,
                          pNumberFormat, dwNumberFlags);
 }
+
+/*************************************************************************
+ *                PathIsEqualOrSubFolder (SHELL32.755)
+ */
+EXTERN_C
+BOOL WINAPI PathIsEqualOrSubFolder(_In_ LPCWSTR pszFile1OrCSIDL, _In_ LPCWSTR pszFile2)
```

Comment:
```suggestion
BOOL WINAPI
PathIsEqualOrSubFolder(
    _In_ LPCWSTR pszFile1OrCSIDL,
    _In_ LPCWSTR pszFile2)
```

Reply:
Also, instead of "File", it's "Path": pszPath1OrCSIDL ; pszPath2 , since they can either represent file paths or directory paths.